### ISSUE TITLE
Feature request: allow overriding a ban from list we don't control / list priorities by our PL?

### ISSUE THREAD

**Issue by Mikaela** [the-draupnir-project/Draupnir#229]
I thought there was an issue about this already, but as not, I will open one.

Fluffynator is subscribed to banlists `fluffy`, `cme-bans` and `EBL`. `ebl` recently banned `@anti-scam:matrix.org` for a reason `not needed bot`, while I would still like to benefit from it pinging for mod attention when there is something suspicious going on as I cannot monitor the rooms 24/7.

I think this might need a different type of allowing users unless `"org.matrix.mjolnir.allow` just works and ability to prioritise policy lists over each other.

As a not-coder I think a simple way of sorting would be looking at the bots PL in the rooms, which would mean `fluffy` leads by priority 100, followed by cme-bans with priority 50 (I didn't check) and at last ebl by priority 0. In the future more flexible controls would likely be desirable.

**Comment by Gnuxie** [comment]
Am I right in clarifying that the problem is that you want to exclude one, or a very few number of bans from `ebl` but otherwise watch all policies directly?

**Comment by Mikaela** [comment]
Yes. If I didn't want the majority of entries applied, I wouldn't have subscribed the list.
